src/main/java/com/coveros/selenified/utilities/ReportOverview.java

```diff
@@ -251,13 +253,15 @@ private void writeResults(List<ClassResult> classResults, String status, String
     private void writeTestResult(String status, String cssClass, String className, ITestResult iTestResult) {
         Browser browser = (Browser) iTestResult.getAttribute(BROWSER);
         Reporter reporter = (Reporter) iTestResult.getAttribute(REPORTER);
-        String link = "--";
+        StringBuilder link = new StringBuilder("--");
         if (reporter != null && !"Skip".equals(status)) {
-            String htmlFilename = reporter.getFileName() + ".html";
-            link = LINK_START + getReportDir(iTestResult) + "/" + htmlFilename + LINK_MIDDLE + "HTML Report" + LINK_END;
+            link = new StringBuilder(LINK_START + getReportDir(iTestResult) + "/" + reporter.getFileName() + ".html" + LINK_MIDDLE + "HTML" + LINK_END);
             if (Property.generatePDF()) {
                 String pdfFilename = reporter.getFileName() + ".pdf";
-                link += " " + LINK_START + getReportDir(iTestResult) + "/" + pdfFilename + LINK_MIDDLE + "PDF Report" + LINK_END;
+                link.append(" " + LINK_START).append(getReportDir(iTestResult)).append("/").append(pdfFilename).append(LINK_MIDDLE).append("PDF").append(LINK_END);
+            }
+            for(Map.Entry<String, LogEntries> log : reporter.getLogs().entrySet()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "log" which hides the field declared at line 56. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AHiddenFieldCheck)